android-components/components/browser/menu2/src/main/java/mozilla/components/browser/menu2/BrowserMenuController.kt

```diff
@@ -37,29 +37,38 @@ class BrowserMenuController(
         notifyObservers { onDismiss() }
     }
 
-    /**
-     * @param anchor The view on which to pin the popup window.
-     * @param orientation The preferred orientation to show the popup window.
-     * @param forceOrientation When set to true, the orientation will be respected even when the
-     * menu doesn't fully fit.
-     */
+    private var orientation: Orientation? = null
+
     override fun show(
         anchor: View,
         orientation: Orientation?,
```

Comment:
Is there a reason for orientation being nullable? To me, it seems like this could cause unpredictable behaviour or a bug.

Reply:
Looking at [the patch](https://github.com/rocketsroger/android-components/commit/87c168eab46ea1570aeabe029e48af3b18331e39) I don't see a reason why it should be null. I think we could make it not nullable as a follow up. Pinging the author, maybe there was a not-so-obvious reason
 @rocketsroger